REST API and UI Collaboration/src/GodelTech.Microservices.WebAndApiCollaboration.IdentityServer/GodelTech.Microservices.WebAndApiCollaboration.IdentityServer.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <OutputType>Exe</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="GodelTech.Microservices.Core" Version="1.3.0" />
+    <PackageReference Include="IdentityModel" Version="4.4.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="IdentityServer4">
+      <HintPath>C:\Users\p.lapatsin\Downloads\IdentityServer4-main\IdentityServer4-main\src\IdentityServer4\src\obj\Debug\netcoreapp3.1\IdentityServer4.dll</HintPath>
```

Comment:
These lines make build to fail on my computer. Project file can be fixed as follows:
```xml
<Project Sdk="Microsoft.NET.Sdk.Web">

  <PropertyGroup>
    <TargetFramework>netcoreapp3.1</TargetFramework>
    <OutputType>Exe</OutputType>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="GodelTech.Microservices.Core" Version="1.3.0" />
    <PackageReference Include="IdentityModel" Version="4.4.0" />
    <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
    <PackageReference Include="IdentityServer4" Version="4.0.4" />
  </ItemGroup>

</Project>

```

Reply:
Sorry for this. I investigated how Identity server works and forgot revert this. It is fixed now.